links-testing/Cargo.toml

```diff
@@ -18,7 +18,7 @@ aws-lc-fips-sys = { path = "../aws-lc-fips-sys", optional = true }
 
 [build-dependencies]
 cc = "1"
-toml_edit = "0.21"
+toml_edit = "0.22"
```

Comment:
[0.19.8](https://crates.io/crates/toml_edit/versions) appears to be the last release supporting our MSRV (1.61) -- however, this crate is not published so it's not relevant.